understanding/22/accessible-authentication-enhanced.html

```diff
@@ -70,15 +70,16 @@ <h2>Intent of Accessible Authentication (Enhanced)</h2>
     <section id="benefits">
       <h2>Benefits of Accessible Authentication (Enhanced)</h2>
       
-        <p>The benefits of this success criterion are the same as <a href="accessible-authentication.html#benefits">Accessible Authentication</a>.</p>
+        <p>The benefits of this success criterion are the same as <a href="accessible-authentication-minimum.html#benefits">Accessible Authentication (Minimum)</a>.</p>
 
       <p>People with cognitive issues relating to memory, reading (for example, dyslexia), numbers (for example, dyscalculia), or perception-processing limitations will be able to authenticate irrespective of the level of their cognitive abilities.</p>
       
    </section>
    
    <section id="examples">
       <h2>Examples of Accessible Authentication (Enhanced)</h2>
-      <p>The examples of this Success Criterion are the same as the <a href="accessible-authentication.html#examples">Accessible Authentication</a> examples.</p>
+      <p>The examples of this success criterion are very similar to the <a href="accessible-authentication-minimum.html#examples">Accessible Authentication (Minimum)</a>.</p>
```

Comment:
Stylistically, "Success Criterion" has been put in upper case, so I'm not sure why you're setting this lc.
```suggestion
      <p>The examples of this Success Criterion are very similar to the <a href="accessible-authentication-minimum.html#examples">Accessible Authentication (Minimum)</a>.</p>
```

Reply:
I don't recall actively making that lc change. wonder if this was lc in the original that i worked on, and the capitalisation came after this PR was filed (and somehow munged when main was merged back into this PR)